src/parser/parser.yy

```diff
@@ -557,6 +568,65 @@ attribute_expression
     }
     ;
 
+case_expression
+    : simple_case_expression {
+        $$ = $1;
+    }
+    | generic_case_expression {
+        $$ = $1;
+    }
+    | conditional_expression {
```

Comment:
Is this part of case_expression?

Reply:
Yes

Reply:
But I don't see this in the document of case expression.

Reply:
It's here: 
![image](https://user-images.githubusercontent.com/38901892/97126804-d90ab400-1772-11eb-9606-b5909b2bf60d.png)
conditional_expression is newly added in nebula. Cypher doesn't hava it.